raceSystem/webUI.py

```diff
@@ -0,0 +1,194 @@
+                try:
+                    countdown = int(request.form['countdown'])
+                except ValueError:
+                    return "<p>Countdown: ongeldige waarde</p>"
+
+                try:
+                    orange = int(request.form['orange'])
+                except ValueError:
+                    return "<p>Oranje: ongeldige waarde</p>"
+
+                if orange >= countdown:
+                    return "<p>Oranje: waarde moet lager zijn dan countdown</p>"
+
+                self.setCountdownCallback(countdown)
+                self.setOrangeLightAtCallback(orange)
+
+                return render_template(
+                    "settings.html",
+                    countdownValue=countdown,
+                    orangeValue=orange)
+            else:
+                countdown = self.getCountdownCallback()
+                orange = self.getOrangeLightAtCallback()
+                return render_template(
+                    "settings.html",
+                    countdownValue=countdown,
+                    orangeValue=orange)
+                pass
+
+    def setCallbackFunctions(
+            self,
+            exitCallback,
+            startRaceCallback,
+            stopRaceCallback,
+            setNameCallback,
+            getNameCallback,
+            setCountdownCallback,
+            getCountdownCallback,
+            setOrangeLightAtCallback,
+            getOrangeLightAtCallback,
+            setResetTimerSeconds,
+            getResetTimerSeconds):
+        self.exitCallback = exitCallback
+        self.startRaceCallback = startRaceCallback
+        self.stopRaceCallback = stopRaceCallback
+        self.setNameCallback = setNameCallback
+        self.getNameCallback = getNameCallback
+        self.setCountdownCallback = setCountdownCallback
+        self.getCountdownCallback = getCountdownCallback
+        self.setOrangeLightAtCallback = setOrangeLightAtCallback
+        self.getOrangeLightAtCallback = getOrangeLightAtCallback
+        self.setResetTimerSecondsCallback = setResetTimerSeconds
+        self.getResetTimerSecondsCallback = getResetTimerSeconds
+
+    def exit(self):
+        """Callback function that will be called from UnicycleRaceSystem"""
+        pass
+
+    def startClientConnected(self):
+        self._startClientConnected = True
+        """Callback function that will be called from UnicycleRaceSystem"""
+        pass
+
+    def finishClientConnected(self):
+        self._finishClientConnected = True
+        """Callback function that will be called from UnicycleRaceSystem"""
+        pass
+
+    def startClientLost(self):
+        self._startClientConnected = False
+        """Callback function that will be called from UnicycleRaceSystem"""
+        pass
+
+    def finishClientLost(self):
+        self._finishClientConnected = False
+        """Callback function that will be called from UnicycleRaceSystem"""
+        pass
+
+    def countDownStarted(self):
```

Comment:
Can you also implemented a max race lenght of 15 seconds here, just like in the terminalUI?

Reply:
Done, exactly as requested.